management/azure_mgmt_authorization/lib/profiles/common/configurable.rb

```diff
@@ -0,0 +1,73 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+module Azure::ARM
+  # The Azure::ARM::Configurable module provides basic configuration for Azure ARM activities.
+  module Configurable
```

Comment:
Having `Default` & `Configurable` both under `Azure::ARM` namespace for all the gems seems like we may run into unexpected behaviors but I am not sure what but if you have verified then good.

Reply:
Already verified. No issues